src/_includes/layouts/case-study.njk

```diff
@@ -18,12 +18,19 @@ layout: base
     <a href="{{ sponsor.website }}" target="_blank" rel="noreferrer noopener">
       <img src="{{ sponsor.logo }}" alt="{{ sponsor.name }}" style="width: 50px;" />
     </a>
-    <p>{{ 'this_case_study_is_sponsored_by' | i18n }} {{ sponsor.name }}.</p>
-    <a href="{{ sponsor.website }}">{{ 'find_out_more' | i18n }}</a>
+    <p class="p">{{ 'this_case_study_is_sponsored_by' | i18n }} {{ sponsor.name }}.</p>
+
+    <details class="tooltip">
+      <summary class="a tooltip__toggle">{{ 'find_out_more' | i18n }}</summary>
+      <div class="tooltip__content">
+        <p class="mb-0 p">Disclaimer about how sponsored case studies work and who is able to sponsor / publish them. Join the Ceph Foundation.</p>
```

Comment:
I think the intention here is "Join  the Ceph Foundation" will link to `{{ locale }}/foundation/join`

Reply:
Ah right, I didn't realise. I've this link back into the tooltip now.